### PR TITLE
JIRA 001

### PR DESCRIPTION
**Pull request by analegaonkar**
review

### CODE REVIEW

File: dbt_project.yml [hunk 1]
```diff
@@ -2,7 +2,7 @@
 # Name your project! Project names should contain only lowercase characters
 # and underscores. A good package name should reflect your organization's
 # name or the intended use of these models
-name: 'my_new_project'
```
Comment (rahulsingh336) [inline dbt_project.yml:5]: why project name is changed?

File: models/core/Generic/dim_dept_tbl.sql [hunk 2]
```diff
@@ -27,7 +27,7 @@ final as
     select d.deptid_key
         , d.DEPT_ID
         , d.EFFECTIVE_DATE
-        , d.EFFECTIVE_STAUS
```
Comment (rahulsingh336) [inline models/core/Generic/dim_dept_tbl.sql:30]: why changing this?

**Review by rahulsingh336 — CHANGES_REQUESTED** [review]
given comments

File: dbt_project.yml (hunk 1 above)
Comment (analegaonkar) [reply dbt_project.yml:5]: :) meow

File: models/core/Generic/dim_dept_tbl.sql (hunk 2 above)
Comment (analegaonkar) [reply models/core/Generic/dim_dept_tbl.sql:30]: uff

**Review by rahulsingh336 — APPROVED** [review]

**Pull request merged**